src/bat/output.rs

```diff
@@ -79,7 +80,27 @@ impl OutputType {
                 let mut process = if is_less {
                     let mut p = Command::new(&pager_path);
                     if args.is_empty() || replace_arguments_to_less {
-                        p.args(vec!["--RAW-CONTROL-CHARS", "--no-init"]);
+                        p.args(vec!["--RAW-CONTROL-CHARS"]);
+
+                        // Passing '--no-init' fixes a bug with '--quit-if-one-screen' in older
+                        // versions of 'less'. Unfortunately, it also breaks mouse-wheel support.
+                        //
+                        // See: http://www.greenwoodsoftware.com/less/news.530.html
+                        //
+                        // For newer versions (530 or 558 on Windows), we omit '--no-init' as it
+                        // is not needed anymore.
+                        match retrieve_less_version() {
+                            None => {
+                                p.arg("--no-init");
+                            }
+                            Some(version)
+                                if (version < 530 || (cfg!(windows) && version < 558)) =>
+                            {
+                                p.arg("--no-init");
+                            }
+                            _ => {}
+                        }
```

Comment:
@lzybkr Would you be able to review this change from the Windows point-of-view? The code here is copied from bat. If Delta simply does not work correctly with anything other than your version of less then perhaps this version-checking doesn't make sense on Windows?